Formula/s/sysdig.rb

```diff
@@ -2,17 +2,21 @@ class Sysdig < Formula
   desc "System-level exploration and troubleshooting tool"
   homepage "https://sysdig.com/"
   license "Apache-2.0"
-  revision 15
 
   stable do
-    url "https://github.com/draios/sysdig/archive/refs/tags/0.38.1.tar.gz"
-    sha256 "68085ea118a4209dbde8f1b75584f9f84610b5856e507ffb0703d8add6331132"
+    url "https://github.com/draios/sysdig/archive/refs/tags/0.40.1.tar.gz"
+    sha256 "f4d465847ba8e814958b5f5818f637595f3d78ce93dbc3b8ff3ee65a80a9b90f"
 
     # Update to value of FALCOSECURITY_LIBS_VERSION with
     # VERSION=#{version} && curl -fsSL https://raw.githubusercontent.com/draios/sysdig/$VERSION/cmake/modules/falcosecurity-libs.cmake | grep -o 'set(FALCOSECURITY_LIBS_VERSION "[0-9.]*")' | awk -F'"' '{print $2}'
     resource "falcosecurity-libs" do
-      url "https://github.com/falcosecurity/libs/archive/refs/tags/0.17.2.tar.gz"
-      sha256 "5c4f0c987272b7d5236f6ab2bbe3906ffdaf76b59817b63cf90cc8c387ab5b15"
+      url "https://github.com/falcosecurity/libs/archive/refs/tags/0.20.0.tar.gz"
+      sha256 "4ae6ddb42a1012bacd88c63abdaa7bd27ca0143c4721338a22c45597e63bc99d"
+
+      livecheck do
+        url "https://github.com/falcosecurity/libs.git"
+        regex(/^v?(\d+(?:\.\d+)+)$/i)
+      end
```

Comment:
I don't think we always want the latest tag – the instructions above the resource block state that we should be checking `https://raw.githubusercontent.com/draios/sysdig/#{LATEST_VERSION}/cmake/modules/falcosecurity-libs.cmake` for the right version.